new-lamassu-admin/src/utils/bill-options.js

```diff
@@ -0,0 +1,12 @@
+import * as R from 'ramda'
+
+const getBillOptions = R.curry((locale, denomiations) => {
+  const currency = R.prop('fiatCurrency')(locale)
+  return R.compose(
+    R.map(code => ({ code: parseInt(code), display: code })),
+    R.keys,
+    R.path([currency])
```

Comment:
I don't think ramda is null safe, does this work even when picking a fiat that has no bill definition?

Reply:
I tested with USD and COP, with and without bill definitions, respectively, and everything seems to work correctly. When there's no bill definitions, it returns an empty array and allows the user to insert a number